tests/import/test_import_cli.py

```diff
@@ -164,35 +161,44 @@ def test_max_candidates_custom(self, tmp_config):
             tmp_db=True,
         )
 
-        mock_new_album = Mock()
         mock_candidates = [Mock() for _ in range(10)]
 
         with patch(
             "moe.moe_import.import_cli.candidate_prompt", autospec=True
         ) as mock_candidate_prompt:
             config.CONFIG.pm.hook.process_candidates(
-                new_album=mock_new_album,
+                new_album=ANY,
                 candidates=mock_candidates,
             )
 
-        mock_candidate_prompt.assert_called_once_with(
-            mock_new_album, mock_candidates[:3]
-        )
+        mock_candidate_prompt.assert_called_once_with(ANY, mock_candidates[:3])
 
     def test_max_candidates_more_than_available(self):
         """Should handle cases where max_candidates > available candidates."""
-        mock_new_album = Mock()
         mock_candidates = [Mock() for _ in range(2)]
 
         with patch(
             "moe.moe_import.import_cli.candidate_prompt", autospec=True
         ) as mock_candidate_prompt:
             config.CONFIG.pm.hook.process_candidates(
-                new_album=mock_new_album,
+                new_album=ANY,
                 candidates=mock_candidates,
             )
 
-        mock_candidate_prompt.assert_called_once_with(mock_new_album, mock_candidates)
+        mock_candidate_prompt.assert_called_once_with(ANY, mock_candidates)
+
+    def test_max_candidates_invalid_value(self, tmp_config):
+        """The max_candidates option should raise an error if set to less than 1."""
+        with pytest.raises(ConfigValidationError):
+            tmp_config(
+                settings="""
+                default_plugins = ['cli', 'import']
+
+                [import]
+                max_candidates = 0
+                """,
+                tmp_db=True,
```

Comment:
We don't need to create a database here (same for test_max_candidates_option) so this can be removed and it should speed up the test a bit.